api/src/main/resources/workbench.yaml

```diff
@@ -967,6 +995,59 @@ definitions:
         format: int64
         description: Milliseconds since the UNIX epoch.
 
+  MaterializeCohortRequest:
+    type: object
+    properties:
+      cohortName:
+        description: >
+         The name of a cohort that is to be evaluated. Either this or cohortSpec should be specified
+        type: string
+      cohortSpec:
+        description: >
+          JSON representation of a cohort to be evaluated (using the same format used for saved
+          cohorts). Either this or cohortName should be specified
+        type: string
+      statusFilter:
+        description: >
+          An array of status values; participants with these statuses will be included.
+          Defaults to [NOT_REVIEWED, INCLUDED, NEEDS_FURTHER_REVIEW] -- everything but EXCLUDED.
+          Only valid for use with cohortName (cohorts saved in the database.)
+        type: array
+        items:
+          $ref: "#/definitions/CohortStatus"
+      cdrVersionName:
```

Comment:
Is there a difference between cdrVersionName and cdrVersionId?

Reply:
cdrVersionId is a numeric identifier which we generate and isn't going to be human readable or visible in our UI -- its our database ID.

cdrVersionName was intended to be the visible name of the CDR version -- e.g. "AllOfUs Registered Q32018". The idea being that somebody could specify this in a notebook without having to retrieve the database ID for the CDR version first.

We could accept cdrVersionId, too; not sure if we need to, though.